Jenkinsfile_nightly

```diff
@@ -40,6 +40,8 @@ def secrets = [
     secret('robotics-email-scottish-to', 'ROBOTICS_EMAIL_SCOTTISH_TO'),
     secret('robotics-email-pip-ae-to', 'ROBOTICS_EMAIL_PIP_AE_TO'),
     secret('AppInsightsInstrumentationKey', 'APPINSIGHTS_INSTRUMENTATIONKEY'),
+    secret('fortify-on-demand-username','FORTIFY_USER_NAME'),
+    secret('fortify-on-demand-password','FORTIFY_PASSWORD')
```

Comment:
These lines should be removed. The secrets should be set on sscs-aat key vault.

Reply:
Removed